packages/luna/src/components/Loading/Loader.tsx

```diff
@@ -19,32 +19,36 @@ export const Loader: React.FC<{
           ease: "linear",
         }}
         style={{
-          maskImage:
-            variant === "large"
-              ? "radial-gradient(circle, transparent 30px, white 16px)"
-              : "radial-gradient(circle, transparent 25px, white 0px)",
+          maskImage: `radial-gradient(circle, transparent 
+          ${variant === "large" ? "30px" : ""}
+				  ${variant === "medium" ? "25px" : ""}
+				  ${variant === "small" ? "15px" : ""}, white
+				  ${variant === "large" ? "16px" : ""}
+				  ${variant === "medium" ? "0px" : ""}
+          ${variant === "small" ? "0px" : ""}
```

Comment:
we can have a function that returns the size based on the variant. 

```
const size = useMemo(() => {
 if (variant === 'large') return '30px';
// ... 
}, [])

// or even better 

```ts
const styles = useMemo(() => {
   if (variant === 'large') return ['30px', '16px'];
   // ... 
}, [])
```